<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>

  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="original">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>ORIGINAL</string>
    </property>
   </widget>

   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>179</y>
      <width>251</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="sliderLayout"/>
   </widget>
   <widget class="QPushButton" name="regenMatrix">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>720</y>
      <width>211</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 90, 214)</string>
    </property>
    <property name="text">
     <string>Regenerate Matrix</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>1011</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="row1"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>1011</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="row2"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>1011</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="row3"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>480</y>
      <width>1011</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="row4"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>600</y>
      <width>1011</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="row5"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>1011</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="x_label_row"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>140</y>
      <width>160</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="y_label_col"/>
   </widget>
   <widget class="QPushButton" name="resetSlidersButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>249</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(90, 214, 158)</string>
    </property>
    <property name="text">
     <string>Reset Sliders</string>
   <widget class="QLabel" name="aug_11">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>1-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_12">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>1-2</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_13">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>120</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>1-3</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_14">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>120</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>1-4</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_15">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>120</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>1-5</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_25">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>240</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>2-5</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_22">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>2-2</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_23">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>240</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>2-3</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_24">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>240</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>2-4</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_21">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>2-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_35">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>360</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>3-5</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_32">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>3-2</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_33">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>360</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>3-3</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_44">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>480</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>4-4</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_41">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>480</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>4-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_34">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>360</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>3-4</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_43">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>480</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>4-3</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_42">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>480</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>4-2</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_31">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>3-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_45">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>480</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>4-5</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_53">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>600</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>5-3</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_51">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>600</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>5-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_52">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>600</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>5-2</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_55">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>600</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>5-5</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_54">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>600</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>5-4</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
